woocommerce/woocommerce-ios
# [Mobile Payments] Avoid loss of cents on receipt totals

Fixes #4434 

To Test:
- Add an item with a price ending in something other than .00 to your cart and order it
- Proceed to pay for it with card present payments
- Ensure the correct amount is shown in both printed and email receipts
- Avoid using prices that trigger payment declines: https://stripe.com/docs/terminal/testing#physical-test-card

Also considered:
- I would have also liked to have added a unit test for this, but that would have required investment in HTML parsing and since we are exploring other options for receipt rendering right now, I elected not to,

Before:

<img src="https://user-images.githubusercontent.com/1595739/122295780-7c4d3c80-ceae-11eb-90ef-1e31615e1558.png" width=50% />

After:

<img src="https://user-images.githubusercontent.com/1595739/122295777-7bb4a600-ceae-11eb-8989-51ccf1577a7e.png" width=50% />

<img src="https://user-images.githubusercontent.com/1595739/122295772-79eae280-ceae-11eb-9f2e-5f005c161bf0.png" width=50% />

Update release notes:
